src/kernel/filesystem/vfs.zig

```diff
@@ -372,6 +394,17 @@ pub fn mount(dir: *DirNode, fs: *const FileSystem) MountError!void {
     dir.mount = fs.getRootNode(fs);
 }
 
+///
+/// Unmount the the filesystem attached to a directory. Any opened files or directories will be
```

Comment:
One too many "the" here. Why does this cause opened files within the mounted dir to be left dangling?

Reply:
All the the's. Yes this is now lie, I has this before I changed how close works, can remove the last part of the doc comment.